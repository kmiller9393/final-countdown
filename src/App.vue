<template>
  <div id="app">
    <ImageContainer />
  </div>
</template>

<script>
import ImageContainer from './components/ImageContainer.vue';

export default {
  name: 'app',
  components: {
    ImageContainer
  }
};
</script>

<style>
html {
  background-color: navy;
}
#app {
  font-family: 'Avenir', Helvetica, Arial, sans-serif;
  -webkit-font-smoothing: antialiased;
  -moz-osx-font-smoothing: grayscale;
  text-align: center;
  color: #2c3e50;
  margin-top: 60px;
}
</style>
