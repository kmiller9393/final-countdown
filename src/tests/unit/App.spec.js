import App from '../../App.vue';
import { shallowMount } from '@vue/test-utils';

describe('App.vue', () => {
  it('should match the snapshot', () => {
    const wrapper = shallowMount(App);
    expect(wrapper).toMatchSnapshot();
  });
});
