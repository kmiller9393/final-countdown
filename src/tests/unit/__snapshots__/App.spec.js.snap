// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.vue should match the snapshot 1`] = `
VueWrapper {
  "_emitted": Object {},
  "_emittedByOrder": Array [],
  "isFunctionalComponent": undefined,
}
`;
